corehq/apps/hqwebapp/static/hqwebapp/js/main.js

```diff
@@ -325,7 +325,7 @@ var COMMCAREHQ = (function () {
 function setUpIeWarning() {
     'use strict';
     var $warning;
-    if ($.browser.msie) {
+    if (/msie/i.test(navigator.userAgent) || /windows/i.test(navigator.userAgent)) {
```

Comment:
Apparently IE comments aren't supported after IE9 - is it only older versions of IE that we're concerned with?